### DISCUSSION TITLE
How to change the physiological scaling factors in septic ICU patients

### DISCUSSION THREAD

**Discussion by Xiaoyang-svg** [Open-Systems-Pharmacology/Forum#1392] [How-Tos]
Dear Community,

Recently, I read an interesting paper that used pksim to develop a  PBPK model of vanomycin in critically ill septic patients (doi: 10.1007/s40262-016-0475-3.), altering the physiological scaling factors associated with different septic states. 

Since my modeling target population is also ICU patients, I want to try to model the population in this way. However, the only parameters I can find in pksim are only hematocrit and total liver blood flow, so I wonder where I can find or how I can change the other parameters stated in the paper (albumin, a1-acid glycoprotein, creatinine clearance, total body water, exttracellular water, intracellular water, total body fat, and total body protein) ?

Looking forward to your reply!
Thanks a lot.

![image](https://user-images.githubusercontent.com/102672543/204118712-b03af003-bc74-4a70-ae22-a21be3d60612.png)

**Comment by prvmalik** [comment]
You can view the parameters listed that were changed in the text by creating a population, then exporting the population to .csv. Vf(water), Vf(protein) etc. are available for fat and muscle.  That allows you to manipulate the population parameters with a software that you desire such as excel, R, MATLAB etc. and then re-upload to PK-Sim. However, this method written in this paper is extremely complex and not specific enough for clear execution, to my understanding.

**Reply by Scorrt542** [reply]
Hello, have you solved the problem of ，Are there any specific steps?

**Reply by Xiaoyang-svg** [reply]
Hi Scorrt, this issue has actually not been resolved and I am no longer working on this project. As far as I know, we could creat a population in pksim and export the population to .csv, and then edit it and reload this file into pksim. However, the various  parameters listed in the paper are not shown in the .csv file, and I don't know how to add theses characteristic parameters in the .csv file and how to reloaded this to pksim again, and also I am not sure whether the changed parameters can be recognized by pksim. So I guess it may be solved when linked to other programming software like Matlab, but I am not good at programming. Sorry for not being able to help you.

**Comment by BBBIMAX** [comment]
Hello, is the question resolved? I also saw this article, and the target population for my study was critically ill patients in the ICU, with important relationships between pharmaceuticals and plasma albumin. Would you like to find out how to set up the albumin content in a patient's plasma?

**Reply by StephanSchaller** [reply]
Yes, this is resolved. YOu can use the instructions from the paper or the simplified approach as @prvmalik mentioned.
If you export your PK-Sim model to MoBi, you can edit all parameters (or through export to PKML and R).
For the latter, please refer to the [documentation and introduction to R](https://www.open-systems-pharmacology.org/OSPSuite-R/)

**Reply by Scorrt542** [reply]
Hello, have you solved the problem of ?

**Reply by BBBIMAX** [reply]
Sorry, I'm no longer working on this research and haven't addressed the above further.

**Comment by Scorrt542** [comment]
Hello, have you solved the problem of ?
